frontend/src/lib/utils.tsx

```diff
@@ -377,9 +377,8 @@ export function stripHTTP(url: string): string {
 export function isURL(string: string): boolean {
     if (!string) return false
     // https://stackoverflow.com/questions/3809401/what-is-a-good-regular-expression-to-match-a-url
-    const expression = /^\s*https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/gi
-    const regex = new RegExp(expression)
-    return !!string.match(regex)
+    const regexp = /^\s*https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/gi
+    return !!string.match?.(regexp)
```

Comment:
Q: Why do we need the `?`? In this context string is not null right?

Reply:
If the signature is lying (this is called with null/undefined) I'd say it's also worth updating the method signature.

Reply:
@Twixes, I agree with @macobo here! If the argument with the name `string` can be either a `string`, `undefined` or `null`, we should specify all the options in the function signature.